src/api/github/__mocks__/getClient.ts

```diff
@@ -56,7 +56,10 @@ function mockClient() {
     },
     teams: {
       getByName: jest.fn(async function (payload) {
-        if (payload.team_slug === 'test' || payload.team_slug === 'rerouted') {
+        if (
+          payload.team_slug === 'product-owners-test' ||
+          payload.team_slug === 'product-owners-rerouted'
```

Comment:
is it more clear to change this to `product_owner_team_slug`? since there are different types of teams now

Reply:
This is driven by GitHub's API design. Where we call this in `route.ts` I prefixed with `gh` to clarify.